README.md

```diff
@@ -141,6 +142,22 @@ add it to your $PATH.
 
 ## Usage
 
+Usage `WSL (Windows Subsystem for Linux)` can be perfect to run this scripts, otherwise you can only modify the script after using `Git Bash` to run.
+So you'd better install and use `WSL` to run scripts.
+
+#### Three ways:
+1.  `Win + R` Combination key, Open `Run` window.
+    Enter & Run `bash` command.
+    Enter & Run `C:\Program Files\Docker Toolbox\start.sh` in `bash` window.
+    
+2.  Change `export PATH="$PATH:/mnt/c/Program Files (x86)/VMware/VMware Workstation"` to `export PATH="$PATH:/c/Program Files (x86)/VMware/VMware Workstation"` in `start.sh`.
+    Find `Docker Quickstart Terminal` on the desktop and double-click to run(need installed `Git Bash`).
+    
+3.  Change `export PATH="$PATH:/mnt/c/Program Files (x86)/VMware/VMware Workstation"` to `export PATH="$PATH:/c/Program Files (x86)/VMware/VMware Workstation"` in `C:\Program Files\Docker Toolbox\start.sh`.
```

Comment:
Both for 2. and 3. could we break the `export [...]` commands to multiple lines to make it more readable?

Reply:
I dont really see the difference between 2. and 3. could you explain?